documentation/src/main/asciidoc/ch01.asciidoc

```diff
@@ -26,18 +26,18 @@ your __pom.xml__:
 ----
 ====
 
-This transitively pulls in the dependency to the Bean Validation API
-(`javax.validation:validation-api:{bvVersion}`).
+This transitively pulls in the dependency to the Jakarta Bean Validation API
+(`jakarta.validation:jakarta.validation-api:{bvVersion}`).
 
 [[validator-gettingstarted-uel]]
 ==== Unified EL
 
-Hibernate Validator requires an implementation of the Unified Expression Language
-(http://jcp.org/en/jsr/detail?id=341[JSR 341]) for evaluating dynamic expressions in constraint
+Hibernate Validator requires an implementation of Jakarta Expression Language
```

Comment:
maybe instead of removing the link we could change to https://projects.eclipse.org/projects/ee4j.el ? I see that later you've used such link for CDI.